Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM almalinux:8 AS stage0
+ARG ALMA_RELEASE=8
```

Comment:
I'm actually unsure if Renovate can detect this... I would rather just be explicit and specify `almalinux:8` since it's only two occurrences.